addons/OPT/Beam/fn_openDialog.sqf

```diff
@@ -1,8 +1,8 @@
 #include "macros.hpp"
 #include "\opt_core\dialogs\OPTDef.hpp"
 
-private _success = createDialog "opt_beamDialog";
 
+private _success = createDialog "opt_beamDialog";
```

Comment:
Wenn das ein fest stehender Name ist, würde ich das als "Konstante" (aka Makro) in ein define auslagern

Reply:
du meinst opt_beamDialog? Das ist der Klassenname des Dialogs. Was ist der Vorteil, das auszulagern?

Reply:
Ja meine ich.
Da kann man ein zentrales define erstellen und das sowohl bei der Definition der Klasse, als auch hier verwenden (bzw. hier bräuchte man eine Version mit Anführungszeichen, aber seis drum).

Wenn man sich später (warum auch immer) dazu entscheidet, dass der Klassenname geändert werden muss, muss man das nur an einer Stelle tun und kann sich sicher sein, damit alle Instanzen dieses Klassennamens geändert zu haben.

Reply:
Hmjoa, ist zwar unwahrscheinlich, tut aber auch nicht weh, hast du Recht.
Dann würde ich sagen, wir fügen den Dialognamen genau wie schon die IDC/IDDs auch einfach der \opt_core\dialogs\OPTDef.hpp hinzu, dann gibt's auch garantiert keine Overlaps.

Reply:
Ja hört sich gut an :+1: